ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -512,11 +512,15 @@ periodics:
               RUN set -xeuo pipefail && \\
                   mkdir -p /srv/repo /tmp/working && \\
                   curl -L "${tar_url}" | tar xf - -C /srv/repo/ && \\
+                  cp /etc/yum.repos.d/fedora-updates-testing-modular.repo /tmp && \\
                   rm -rf /etc/yum.repos.d/* && \\
+                  cp /tmp/fedora-updates-testing-modular.repo /etc/yum.repos.d/ && \\
+                  sed -i 's/enabled=0/enabled=1/g' /etc/yum.repos.d/fedora-updates-testing-modular.repo && \\
                   rpm-ostree db list --repo /srv/repo "${ostree_ref}" > /tmp/packages && \\
+                  dnf module enable -y cri-o:1.16 && \\
```

Comment:
it needs to be crio 1.17 now

Reply:
There hasn't been cri-o 1.17 release yet - and no 1.17 module yet. I'll be pinging @lsm5 to get module created and update this task